### PR TITLE
Adding tooltips for costum columns in scolv

### PR DESCRIPTION
**Pull request by luca-s**
Dear Developers,

I would like to propose this change for the custom column option of scolv. When the custom column is created I would set the tooltip for that column to be the same as the content of the cell.

What do you think about it?

Regards,
Luca

### CODE REVIEW

File: libs/seiscomp/gui/datamodel/eventlistview.cpp [hunk 1]
```diff
@@ -889,11 +889,13 @@ class OriginTreeItem : public SchemeTreeItem {
 
 			if ( config.customColumn != -1 ) {
 				setText(config.customColumn, config.customDefaultText);
+			  setToolTip(config.customColumn, config.customDefaultText);
```
Comment (gempa-jabe) [inline libs/seiscomp/gui/datamodel/eventlistview.cpp:892]: Missing tab

**Comment by gempa-jabe** [comment]
I am fine with adding tooltips to each of the columns. What is your actual intention for that?

**Comment by luca-s** [comment]
In my use case the column (an origin ID and a long text in another case) is too long to be visualised without horizontally scrolling the events table. This is because the table is quite packed with important information already and I do not want to hide any of them. With the tooltip I do not need to do so, I can simply leave the mouse at the beginning of the custom column, which is only partially visible and see the content in the tooltip.

**Comment by gempa-jabe** [comment]
Thank you for clarification. To accept the PR you need to sign the CLA. If you agree just contact me otherwise close the PR and I will apply the changes myself according to your description.

**Comment by luca-s** [comment]
@gempa-jabe sure, no problem for signing the CLA, I'll contact you via email.

**Comment by luca-s** [comment]
@gempa-jabe

> otherwise close the PR and I will apply the changes myself according to your description.

Just for this one, do you mind  applying the patch yourself? thank you very much, I am closing the PR  now

**Pull request closed without merging**